plugins/flatpak/tests/app-update/org.test.Chiron/metadata

```diff
@@ -1,4 +1,4 @@
 [Application]
 name=org.test.Chiron
-runtime=org.test.Runtime/x86_64/master
+runtime=org.test.Runtime/x86_64/new_master
```

Comment:
Nothing has changed in the previous use case, because the error we're fixing of course that it wouldn't care about the runtime in the update.
Changing the branch, as we do often in Endless is actually a better test because we don't give a chance for e.g. checking just the ID instead of the unique ID of apps.